packages/i18n/src/lang/fr.json

```diff
@@ -199,6 +199,10 @@
     "defaultMessage": "aaaa",
     "description": "Placeholder for a year input"
   },
+  "4WvSir": {
+    "defaultMessage": "Afficher tout",
```

Comment:
Based on a previously use string including **Expand all**, this seems inconsistent.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/a66841ef459952763f3b409238d2b2329f42cd00/apps/web/src/lang/fr.json#L2726-L2729

https://github.com/GCTC-NTGC/gc-digital-talent/blob/21cfb0d68aa602ec6b58923aaf3e3a8c5dffb988/apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/components/ApplicationInformation/ApplicationInformation.tsx#L191-L194

Reply:
Fixed in [add fixed translations](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9145/commits/5a0ff8e1dc26025f991a9eec3b98e450d90c3088)